content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -279,10 +280,25 @@ By distributing CPUs evenly across NUMA nodes, application developers can more
 easily ensure that no single worker suffers from NUMA effects more than any
 other, improving the overall performance of these types of applications.
 
+If the `align-by-socket` policy option is specified, CPUs will be considered
+aligned at the socket boundary when deciding how to allocate CPUs to a
+container. By default, the `CPUManager` aligns CPU allocations at the NUMA
+boundary, which could result in performance degradation if CPUs need to be
+pulled from more than one NUMA node to satisfy the allocation. Although it
+tries to ensure that all CPUs are allocated from the _minimum_ number of NUMA
+nodes, there is no guarantee that those NUMA nodes will be on the same socket.
+By directing the `CPUManager` to explicitly align CPUs at the socket boundary
+rather than the NUMA boundary, we are able to avoid such issues. Note, this
+policy option is not compatible with `TopologyManager` `single-numa-node`
+policy and does not apply to hardware where the number of sockets is greater
+than number of NUMA nodes.
```

Comment:
You should add a newline after this

Reply:
Addressed the comment. 
Also squashed the unnecessary commits